README.md

```diff
@@ -60,6 +60,9 @@ _Sponsored link:_ [Front End Developer Jobs](https://www.bootstrapcdn.com/jobs/?
   * [USD](https://exchangeratesapi.io/api/latest?base=USD)
   * [EUR](https://exchangeratesapi.io/api/latest?base=EUR)
 * [VAT rates for EU](http://jsonvat.com/)
+* [Exchange Rate API](https://www.exchangerate-api.com)
```

Comment:
How is this more awesome than exchangeratesapi.io?

Reply:
I've got multiple data sources rather than just the ECB dataset that exchangeratesapi.io uses. This means the dataset can exclude outlier values and support more currency codes - 50 vs 32 at the moment. Otherwise they are pretty similar, so I'd understand if you'd prefer not to merge.

Reply:
Ok remove `exchangeratesapi.io` as well and I'll merge it asap

Reply:
I've included a commit to remove the other listing. Thanks!